go.mod

```diff
@@ -254,6 +255,9 @@ require (
 )
 
 replace (
+	// Replace this import until there's an official release of x/minfee.
+	// After this PR merges, we can create a release of x/minfee v1.0.0
```

Comment:
same, minfee is already at v3 with app, no?

Reply:
We don't have any releases of x/minfee yet so I intended on starting with v1.0.0 instead of v3.0.0